server/webapp/WEB-INF/rails.new/spec/views/admin/users/_user_results_table_html_spec.rb

```diff
@@ -28,4 +28,28 @@
 
     expect(response.body).to have_selector("input[type='radio'][name='selection'][id*='selection_button_user_username_dot_with_dot_dot']")
   end
+
+  it "should santize all fields when generating a DOM ID, to handle fields with special characters" do
+    allow(view).to receive(:random_dom_id) {|prefix| prefix + 'RAND'}
+
+    user_search_model = double('user_search_model')
+    user = User.new('!username-with_special_chars._@#$%^&*()-_ and spaces', '!display-name-with_special_chars_@#$%^&*()-_', "email!me#now*@dot.com")
```

Comment:
Line is too long. [148/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.